cat-theme/src/templates/content/carousel_card.ftl

```diff
@@ -0,0 +1,35 @@
+<section id="carousel_card" class="carousel_card">
+    <div class="container">
+        <div class="row">
+
+            <div class="carousel_card__wrapper">
+                <div class="carousel_card__header col-12 col-sm-4">
+
+                    <h2>Two lines of Text Here</h2>
+
+                </div>
+                <div class="carousel_card__container col-12 col-sm-8">
+
+                    <div class="carousel_card__main active">
+
+                        <div class="carousel_card__main--top">
+                            <p>Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats Cats 
+                            </p>
```

Comment:
We should only use a `<p />` tag if it is wrapping a paragraph for semantic purposes.